test/test_transforms_v2.py

```diff
@@ -186,7 +187,21 @@ def _check_functional_scripted_smoke(functional, input, *args, **kwargs):
         functional_scripted(input.as_subclass(torch.Tensor), *args, **kwargs)
 
 
-def check_functional(functional, input, *args, check_scripted_smoke=True, **kwargs):
+def _check_functional_torch_compile_smoke(functional, input, *args, **kwargs):
+    """Checks if the functional can be torch compiled and the compiled version can be called without error."""
+    if not isinstance(input, tv_tensors.Image):
+        return
+
+    functional_compiled = torch.compile(functional)
+    functional_compiled(input.as_subclass(torch.Tensor), *args, **kwargs)
+
+    explanation = torch._dynamo.explain(functional_compiled)(input.as_subclass(torch.Tensor), *args, **kwargs)
+    # TODO: Set expected values to 1, 0 once fixed the graph break related to function registration
+    assert explanation.graph_count in (1, 2)
+    assert explanation.graph_break_count in (0, 1)
```

Comment:
I think these 2 asserts are redundant, we can probably just assert the break counts?
Also, could we assert the exact number of graph breaks, instead of checking that it's 0 or 1 ?

Reply:
Yes, we can just have one assert `assert explanation.graph_break_count in (0, 1)`. 
Why `assert explanation.graph_break_count in (0, 1)` and not `assert explanation.graph_break_count == 1`, because there are ops with configs such `to_dtype` on f32 and `dtype=torch.float32, scale=False` that there is no graph to build and explanation object reports `explanation.graph_break_count == 0`.

Reply:
I was thinking that we could have this mapping `kernel -> expected_break_count` so that we can more easily track which ones need to be addressed?

Reply:
In this case I have to combine somehow `check_torch_compile_smoke` flag to enable/disable torch compile tests and `expected_break_count`. For example, replace boolean `check_torch_compile_smoke` by `torch_compile_expected_break_count: Optional[int]` (if None, tests are disabled) ?